schema/360-giving-package-schema.json

```diff
@@ -29,7 +29,7 @@
         },
         "issued": {
           "title": "Issued",
-          "description": "The date that this data package was issued.",
+          "description": "The date that this data package was issued. The date must be written as YYYY-MM-DD, or in full date-time format",
```

Comment:
Could we link to [JSON Schema's date-time format](https://json-schema.org/understanding-json-schema/reference/string.html#dates-and-times) ?

Reply:
I think this is reasonable.

Reply:
We don't link to JSON schema's date formatting in any of the other parts of the schema that use this formatting so seems odd to do so in this case only. We have field guidance on date and date-time, so I think that provides sufficient supporting information.